### PR TITLE
Remove rocksmq fix channel name

### PR DESCRIPTION
**Pull request by fishpenguin**
Signed-off-by: fishpenguin <kun.yu@zilliz.com>

/kind enhancement

issue: #12742 

This pr removes the fixChannelName in rocksmq, which will cause not compatible.

### CODE REVIEW

File: internal/util/rocksmq/server/rocksmq/rocksmq_reader.go [hunk 1]
```diff
@@ -38,8 +38,8 @@ type rocksmqReader struct {
 //Seek seek the rocksmq reader to the pointed position
 func (rr *rocksmqReader) Seek(msgID UniqueID) { //nolint:govet
 	rr.currentID = msgID
-	fixChanName, _ := fixChannelName(rr.topic)
-	dataKey := path.Join(fixChanName, strconv.FormatInt(msgID, 10))
+	fixTopicName := rr.topic + "/"
```
Comment (xiaofan-luan) [inline internal/util/rocksmq/server/rocksmq/rocksmq_reader.go:41]: Do we need to ban topicname contains "/" ?

Comment (fishpenguin) [reply internal/util/rocksmq/server/rocksmq/rocksmq_reader.go:41]: I'll add this constraint in CreateTopic in later pr.

**Comment by xiaofan-luan** [comment]
probably need to add more unit test on topic name with "/", overall is good

**Pull request merged**